src/api/app/views/webui2/webui/packages/build_reason/index.html.haml

```diff
@@ -0,0 +1,38 @@
+- @pagetitle = "Build Reason for #{@project} / #{@package_name}"
+
+.card
+  = render partial: 'webui2/webui/package/tabs', locals: { project: @project, package: @package }
+  .card-body
+    %h3
+      = @pagetitle
+    %h6.subtitle
+      Repository / Architecture: #{@repository.name} / #{@architecture}}
+    %p
+      = link_to '<= Go back to binaries',
+      package_binaries_path(controller: '/webui/package', project: @project, package: @package_name, repository: @repository.name)
```

Comment:
I think the `controller: 'webui/package'` is not needed, right? :thinking: :

Reply:
Right, removed!